src/toil/leader.py

```diff
@@ -335,6 +336,10 @@ def mainLoop(config, batchSystem, jobStore, rootJobWrapper):
     logger.info("Found %s jobs to start and %i jobs with successors to run",
                 len(toilState.updatedJobs), len(toilState.successorCounts))
 
+    #Set proper environment variables
+    for v in getEnv():
```

Comment:
I'm slightly confused, should getEnv() be a function in both the batch system and jobstore? if so what is the purpose of AbstractBatchSystem.getEnv()?